daemon/containerd/service.go

```
@@ -62,7 +63,38 @@ func (cs *ImageService) PullImage(ctx context.Context, image, tagOrDigest string
 		}
 	}
 
-	_, err = cs.client.Pull(ctx, ref.String(), opts...)
+	jobs := newJobs()
+	h := containerdimages.HandlerFunc(func(ctx context.Context, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+		if desc.MediaType != containerdimages.MediaTypeDockerSchema1Manifest {
```

Should we return an error for schema1?